src/components/books/BookItem.vue

```diff
@@ -0,0 +1,47 @@
+<template lang="pug">
+a(href='#')
+  .book-item(:class='{ "book-item-small": small }')
+    img.book-item-image(alt='book image' src='' v-bind:src='book.image_url || "assets/book_logo.png"')
```

Comment:
why are you doing this `src=""` why not `:src="book.image_url || "assets/book_logo.png'"` did you try adding a comma before the `:src`??

Reply:
Maybe the alt can be related to the book itself